src/main.rs

```diff
@@ -62,11 +56,12 @@ fn main() {
         } else {
             result.map(|_| ())
         }
-    } else if args.flag_list {
+    } else if args.is_present("list") {
         list_fuzz_targets()
             .map(|_| ())
     } else {
-        println!("Invalid arguments. Usage:\n{}", USAGE);
+        println!("Invalid arguments. Usage:\n{}",
```

Comment:
Ah yeah, it does that already. I added this bit to keep existing behaviour but I'll happily drop it.